todoapp/src/components/Todo.vue

```diff
@@ -42,6 +42,20 @@ function setPriority(todo: TodoType, number: number) {
   todo.priorityChange = false
 }
 
+function markTodoStatus(todo: TodoType) {
+  if (todo.status) {
```

Comment:
I think you should have one function to set the Todo status done or not done, and in the implementation of that function should decide what should happen based on the returned todo status.